bns/getArchivedFormsListFromSheets.js

```diff
@@ -42,21 +59,21 @@ fn(state => {
 
   //   const cursorValue = manualCursor || dynamicCursor;
   //   console.log('Cursor value to use in query:', cursorValue);
-
-  const formsList = sheetsData.map(survey => ({
+  const combinedData = [...deployedData, ...archivedData];
+  const formsList = combinedData.map(survey => ({
     formId: survey.uid,
     tag: survey.tag,
     name: survey.name,
   }));
 
-  console.log('# of archived forms detected in Sheet:: ', formsList.length);
+  console.log('# of forms detected in Sheet:: ', formsList.length);
   console.log(
     'List of forms to re-sync:: ',
     JSON.stringify(formsList, null, 2)
   );
 
   state.data = {
-    surveys: sheetsData.map(survey => ({
+    surveys: combinedData.map(survey => ({
```

Comment:
@mtuchi  Isn't this a repetition of what is on the `formList` on line 63?

Reply:
Kind of but `formList` and `surveys` have different purpose 
L76 has more data `[formId, tag, name, owner, url]` that needs to be sent to a workflow trigger while the `formList` in L63 is for logging purpose and has few data `[formId, tag, name] only`